package.json

```diff
@@ -8,22 +8,20 @@
   },
   "scripts": {
     "prepare": "husky install && yarn build",
-    "clean": "lerna run clean && lerna clean --yes && rm -rf node_modules",
     "lint": "eslint packages/ scripts/ --ext=js,mjs,ts",
     "format": "prettier --write '{packages,scripts}/**/*.{js,mjs,ts,json,md}'",
     "bundlesize": "bundlesize --config ./scripts/bundlesize/bundlesize.config.json",
-    "build": "lerna run build --ignore @lwc/perf-benchmarks --ignore @lwc/perf-benchmarks-components --ignore @lwc/integration-tests",
-    "build:performance": "yarn run build:performance:components && yarn run build:performance:benchmarks",
-    "build:performance:components": "lerna exec --scope @lwc/perf-benchmarks-components -- yarn build",
-    "build:performance:benchmarks": "lerna exec --scope @lwc/perf-benchmarks -- yarn build",
-    "dev": "lerna run dev --parallel --ignore @lwc/perf-benchmarks --ignore @lwc/perf-benchmarks-components --ignore @lwc/integration-tests",
+    "build": "nx run-many --target=build --all --exclude=@lwc/perf-benchmarks,@lwc/perf-benchmarks-components,@lwc/integration-tests",
+    "build:performance": "yarn build:performance:components && yarn build:performance:benchmarks",
+    "build:performance:components": "nx build @lwc/perf-benchmarks-components",
+    "build:performance:benchmarks": "nx build @lwc/perf-benchmarks",
+    "dev": "nx run-many --target=dev --all --parallel=999 --exclude=@lwc/perf-benchmarks,@lwc/perf-benchmarks-components,@lwc/integration-tests",
     "test": "jest --config ./scripts/jest/root.config.js",
     "test:debug": "node --inspect node_modules/.bin/jest --config ./scripts/jest/root.config.js --runInBand --watch",
     "test:ci": "yarn test --no-cache --coverage --runInBand",
-    "test:integration": "lerna exec --scope @lwc/integration-tests -- yarn sauce",
-    "test:performance": "lerna exec --scope @lwc/perf-benchmarks -- yarn test",
-    "release:publish:ci": "./scripts/release/publish.mjs",
-    "release:version": "./scripts/release/version.mjs"
+    "test:integration": "nx sauce @lwc/integration-tests",
+    "test:performance": "nx test @lwc/perf-benchmarks",
```

Comment:
@nolanlawson I'm using the `test` target for the perf benchmarks for the same reason as sauce, to build the dependencies and then build the perf benchmarks/components before running the tests.